src/metrics/encoding/protobuf/reset.go

```diff
@@ -32,13 +32,13 @@ func resetAggregatedMetricProto(pb *metricpb.AggregatedMetric) {
 	pb.EncodeNanos = 0
 }
 
-// resetMetricWithMetadatasProto resets the metric with metadatas proto, and
+// ResetMetricWithMetadatasProto resets the metric with metadatas proto, and
```

Comment:
nit: this comment is kinda hard to understand

Reply:
Agreed, updating.